src/frontend/Felles/Visningskomponenter/InputUtenSpinner.tsx

```diff
@@ -33,20 +33,14 @@ const StyledInputUtenSpinner = styled(FamilieInput)`
 
 const InputUtenSpinner: React.FC<PropsInputUtenSpinner> = ({ formatValue, ...props }) => {
     const [harFokus, settHarFokus] = useState(false);
-    if (!harFokus) {
-        return (
-            <StyledInputUtenSpinner
-                {...props}
-                type="text"
-                autoComplete="off"
-                value={formatValue ? formatValue(props.value) : props.value}
-                onWheel={(event) => event.currentTarget.blur()}
-                label={''}
-                onFocus={() => settHarFokus(true)}
-                hideLabel
-            />
-        );
-    } else {
+
+    const verdi = formatValue ? formatValue(props.value) : props.value;
```

Comment:
Den skal vel kun formatere verdien hvis den IKKE har fokus - det er poenget med denne komponenten tror jeg

Reply:
Aha, så ikke at den ble brukt kun på en av inputene - kanskje `const formatertVerdi` her for å tydeliggjøre den? :)

Reply:
enig i mer presis navngivning: [c454bdf](https://github.com/navikt/familie-ef-sak-frontend/pull/2810/commits/c454bdf37ea8266e122e8610210a132395640709)